app/models/manageiq/providers/google/manager_mixin.rb

```diff
@@ -77,6 +77,9 @@ def verify_credentials(args)
     end
 
     def raw_connect(google_project, google_json_key, options, proxy_uri = nil, validate = false)
+      require "google/apis"
+      ::Google::Apis.logger = $gce_log
```

Comment:
@Fryguy not sure if there's a better place to do this at startup without eager requiring `"google/apis"`

Reply:
This is basically what we do in the other providers, too, isn't it?

Reply:
Well its not ideal that we can't set the logger on the fog::compute object and that it is a "global" setting but not much we can do about that, that's just how fog-google and goolgle-api-client works